src/Uno.UI/LinkerDefinition.Wasm.xml

```diff
@@ -15,6 +15,7 @@
   <assembly fullname="Uno">
     <type fullname="Windows.UI.Core.CoreDispatcher" />
     <type fullname="Windows.UI.Core.SystemNavigationManager" />
+    <type fullname="Windows.Devices.Sensors.Accelerometer" />
```

Comment:
You can specify a linker condition for this method specifically `Windows.Devices.Sensors.Accelerometer:DispatchReading`

Reply:
Done